web-client/iron-remote-gui/src/iron-remote-gui.svelte

```diff
@@ -445,6 +445,11 @@
             scaleSession(s);
         });
 
+        wasmService.dynamicResize.subscribe((evt) => {
+            loggingService.info('Dynamic resize!');
```

Comment:
Leftover?

Reply:
[suggestion]: Maybe add width/height values to this log message?

Reply:
I also want to suggest the `debug` or `trace` level instead of `info`. It’s typically not something we need to log at the `info` level.

Reply:
updated, IronRDP does not have debug logging for now, only info and error.

I'll update to create a level logging in another PR